Source/AIExtensions/Public/AIPerception/AIExtTeamEvent.h

```diff
@@ -0,0 +1,16 @@
+﻿#pragma once
+
+#include "CoreMinimal.h"
+#include "Kismet/BlueprintFunctionLibrary.h"
+#include "UObject/ObjectMacros.h"
+
+#include "AIExtTeamEvent.generated.h"
+
+UCLASS()
+class AIEXTENSIONS_API UAIExtTeamEvent final : public UBlueprintFunctionLibrary
```

Comment:
Should we create a new BP func library just for this?

Reply:
what do you mean?